sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -196,6 +196,8 @@ task integrationTest(type: Test, dependsOn: processTestResources) {
   exclude '**/BigQueryIOStorageWriteIT.class'
   exclude '**/BigQueryToTableIT.class'
   exclude '**/BigQueryIOJsonTest.class'
+  // TODO Firestore IT test needs to run in a separate project from Datastore.
+  exclude '**/FirestoreV1IT.class'
```

Comment:
I have locally run FirestoreV1IT test against a different GCP project to make sure the code behaves correctly. I'm currently working on another change to enable this test, hopefully in the same GCP project by using a new Firestore feature that's not launched yet but we can internally allowlist this test project to use that. I expect to have that ready in a few days, but I don't want it to block this PR because FirestoreV1IT has never run in the past because it was incorrectly written and it's an unrelated problem from read_time support.

Reply:
+1  for enabling the tests soon. But we don't have to block the PR on that.

Reply:
#23322 is out for review and will fix and enable FirestoreV1IT